py/wait_cat.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
```

Comment:
this program is basically the same as `cat` except it will wait around for the files if it doesnt find them on disk right away